rct229/rulesets/ashrae9012019/section5/section5rule15.py

```diff
@@ -109,6 +109,11 @@ def get_manual_check_required_msg(self, context, calc_vals=None, data=None):
             return manual_check_msg
 
         def rule_check(self, context, calc_vals=None, data=None):
+            return not calc_vals["manual_check_flag"] and calc_vals["wwr_b"] == min(
+                calc_vals["wwr_p"], WWR_THRESHOLD
+            )
```

Comment:
This logic can be refined to:
```python
return calc_vals["wwr_b"] == min(calc_vals["wwr_p"], WWR_THRESHOLD)
```

The reason is because `calc_vals["manual_check_flag"]` is flagged in `manual_check_required` function so it is always `False` in the `rule_check` function.